### ISSUE TITLE
Can't use k8s-resource as a bundled resource type due to rootfs export

### ISSUE THREAD

**Issue by gtg3vv** [jgriff/k8s-resource#35]
We're attempting to bundle this resource with our concourse workers, but can't because the kubectl executable relies on the PATH env which gets lost on export to a flat fs. The bundled resource types must be a rootfs.tgz.

The process is described here: https://concourse-ci.org/concourse-worker.html#installing-or-upgrading-bundled-resource-types

But generally to reproduce:
```shell
docker run --entrypoint bash -it bitnami/kubectl
env
PATH=/opt/bitnami/kubectl/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
# kubectl works
```

```shell
docker export $(docker create bitnami/kubectl) | gzip > rootfs.tgz
docker import rootfs.tgz testkubectl
docker run --entrypoint bash -it testkubectl
env
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
# kubectl fails
```

As far as I can tell we'd need to either symlink the kubectl executable to a standard path like /usr/local/bin, or use the full path to the executable in the resource scripts. Thoughts?

```Dockerfile
FROM bitnami/kubectl

USER 0
RUN ln -s /opt/bitnami/kubectl/bin/kubectl /usr/local/bin/kubectl

USER 1001
```

**Comment by jgriff** [comment]
The symlink option seems to be the best way to resolve this.  I've already verified `kubectl` is available on the path after export/import using this approach.

**Comment by jgriff** [comment]
Fixed in #36 and available as of `latest` and `1.1.0`.
